contrib/fedora-default-kernel/default-kernel.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+# get list of surface kernels
+kernels=$(find /boot -maxdepth 1 -name "vmlinuz-*.surface.*")
+
+# sort by version
+kernels=$(echo "${kernels}" | sort)
```

Comment:
This is probably incompatible with your refind support, but maybe we should sort by the timestamp instead? so that we keep the Fedora behaviour of defaulting to the kernel that was installed last (not neccessarily the one with the larger version number).

Reply:
Hmm, we could do that. I think it should even work with the refind support: We're only matching the surface kernels, so the behavior is that on any change, the timestamp of the last installed surface kernel will get updated to be more recent than any installed kernel, so this essentially only prefers surface kernels. And we're not re-ordering the installed surface kernels, because we're only updating the one with the latest timestamp.

Reply:
Done.